app/views/pages/_form.html.haml

```diff
@@ -57,12 +57,12 @@
         = form.label :meta_title
       .form__input-wrapper
         = form.text_field :meta_title, class: 'form__input form__input--has-icon', data: {dough_mirrorinputvalue_target: '', dough_component: 'CharacterCounter', dough_character_counter_max_chars: 70, dough_character_counter_context: 'meta-title'}
-        %span.form__hint
-          %span{data: {dough_character_counter_indicator: true, dough_character_counter_context: 'meta-title'}}
+        %span.form__hint.form__hint--align-right
+          %span.form__character-count-text{data: {dough_character_counter_indicator: true, dough_character_counter_context: 'meta-title'}}
             0
           %span
             characters remaining
-        %span.form__character-count-warning{data: {dough_character_counter_warning: true, dough_character_counter_context: 'meta-title', title: 'You are over the recommended limit of 70 characters, please shorten your text.'}}
+        %span.form__character-count-warning-icon{title: 'You are over the recommended limit of 70 characters, please shorten your text.', data: {dough_character_counter_warning: true, dough_character_counter_context: 'meta-title'}}
```

Comment:
I wonder if we can extract this to a helper. But I'm a little busy right now to do it though. So :+1: and I will see this in another PR.